### PR TITLE
Feature/consistent code style

### LINKED ISSUE #1643: Clicking on "click here" to add API backend on empty dashboard does not work
On nightly. When you have no API backends, there is this text:

> You don't have any API Backend you manage. 
> To add an API Backend, click here.

Clicking on "click here" does nothing.

### LINKED ISSUE #1690: Improve consistency of Handlebars helpers (i18n, partial, block)
Our Handlebars syntax uses a mixture of elements with and without spaces separating the handlebars and the prefix. E.g.

- `{{_` vs. `{{ _`
- `{{#` vs. `{{ #`
- `{{>` vs. `{{ >`

While this does not make a difference to the Handlebars compiler, it has the following shortcomings for humans:

- it is inconsistent
- including spaces marginally reduces readability
- it has caused trouble when doing a project wide search/replace when cleaning up i18n strings

# Proposal
Reformat all, or most, Handlebars code to remove spaces between the handlebars (`{{`) and 'prefix' (`_`, `#`, and `>`).

# Rationale
A quick search of our code found the following counts:
- 139 results for `{{>`
- 22 results for `{{ >`

Similarly, we tend to keep the `#` character next to the `{{` in template block helpers:

- 107 results for `{{#`
- 41 results for `{{ # `

We should prefer the first version, keeping `>` and `#` close to the `{{`  for these reasons:

- both cases are more common in our code, currently
- the `>` and `#` characters are a bit noisy, since they are not English words (i.e. they are more to help the computer than the developer who is reading the sourcecode)
- combining the `{{>` and `{{#` into single string, without space, makes them simpler (both to read and type)

Both the handlebars (`{{`) and prefixes (`_`, `#`, and `>`) are used primarily to help the computer (parser) to do its job.

Combining handlebars and prefixes makes it, slightly, easier for the human to read, and still accomplishes the same. I.e. `{{_`, `{{>` and `{{#` are easy for us to read and distinguish (without spaces), while the parser can still do its job just fine.

## Quick summary
I am proposing we remove the space infix because:

- handlebars spaces are unnecessary
- handlebars spaces do not significantly increase readability (although spaces increase readability elsewhere in our code)
  - i.e. template helpers without spaces between the prefix are easy to read, since the last character is easy to distinguish
- most template helpers in our code *do not* separate the handlebars (`{{`) from the prefix (`_`, `>`, `#`)

### PR DESCRIPTION
**Pull request by brylie**
Closes #1690 
Closes #1643 

Working through our codebase to improve the consistency of:

- handlebars helpers
- template nesting
- adding internationalization strings where missing

### CODE REVIEW

**Comment by brylie** [comment]
@marla-singer please review.

File: apis/client/import_documentation/import.html [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
-               id="apiDocumentationFile"
-               name="apiDocumentationFile"
-               class="input-file"
-               type="file"
-               >
+         id="apiDocumentationFile"
+         name="apiDocumentationFile"
+         class="input-file"
+         type="file"
+         >
```
Comment (marla-singer) [inline apis/client/import_documentation/import.html]: Check the indentation on this fragment. It may be sufficient to use two spaces for attribute enumeration

File: apis/client/view/settings/settings.html [hunk 2]
```diff
@@ -39,19 +43,19 @@ <h2 class="panel-title">
       <p>
         {{_ "apiSettings_visibility_text" }}
       </p>
-      {{#if api.currentUserCanEdit }}
-        {{> visibilityToggle api=api }}
-      {{/if }}
+      {{> visibilityToggle api=api }}
```
Comment (marla-singer) [inline apis/client/view/settings/settings.html]: Why did you remove condition? It was added to don't show toggle to anonymous and non-owner

File: backlog/client/list/list.html [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
-    {{> autoformModals}}
-    {{#if backlogItems}}
-        {{#each item in backlogItems }}
-          {{> backlogItem item=item api=api }}
-        {{/each}}
-    {{else}}
-        <i>{{_ "apiBacklogList_noApiBacklogItemsFound"}}</i>
-    {{/if}}
+  {{> autoformModals }}
+  {{# if backlogItems}}
```
Comment (marla-singer) [inline backlog/client/list/list.html]: Forgot a space after helper

File: catalogue/client/table/table.html [hunk 4]
```diff
@@ -2,8 +2,8 @@
   <div class="table-responsive">
     <table class="table table-striped table-bordered api-catalogue-table">
       <tbody>
-        {{#each api in apis }}
-        {{#if api.currentUserCanView }}
+        {{# each api in apis }}
+        {{# if api.currentUserCanView }}
```
Comment (marla-singer) [inline catalogue/client/table/table.html]: Check the indentation on this file. Condition is nested

File: documentation/client/manage/manage.html [hunk 5]
```diff
@@ -6,19 +6,24 @@
...
-          <h1 class="modal-title">{{_ "manageApiDocumentationModal_Title" }}</h1>
+          <h1 class="modal-title">
+            {{_ "manageApiDocumentationModal_Title" }}
+          </h1>
         </div>
         <div class="modal-body">
-          {{#autoForm collection=apisCollection doc=api id="apiDocumentationForm" type="update" }}
+          {{# autoForm
+            collection=apisCollection
+            doc=api id="apiDocumentationForm"
```
Comment (marla-singer) [inline documentation/client/manage/manage.html]: Separate this line to two lines too

File: feedback/client/form/form.html [hunk 6]
```diff
@@ -12,22 +12,27 @@
...
+          collection=feedbackCollection
+          type="insert" }}
           <div class="modal-body">
-            <p>{{_ "feedback_feedbackForm_description"}}</p>
-            {{> afQuickField name='topic'}}
+            <p>
+              {{_ "feedback_feedbackForm_description" }}
+            </p>
+            {{> afQuickField name='topic' }}
             {{> afQuickField name='message' rows=4}}
```
Comment (marla-singer) [inline feedback/client/form/form.html]: Add a space after number '4'

File: feedback/client/list/list.html [hunk 7]
```diff
@@ -1,36 +1,38 @@
 <template name="feedbackList">
   <div class="panel panel-default">
     <div class="panel-heading">
-      {{ # if currentUser }}
+      {{# if currentUser }}
         <button
           type="button"
           class="btn btn-info btn-xs pull-right feedback-form-button"
           id="add-feedback"
           >
```
Comment (marla-singer) [inline feedback/client/list/list.html]: Return the sign "**>**" on the previous line

File: proxies/client/form/form.html [hunk 8]
```diff
@@ -4,30 +4,35 @@
...
-          <h4 class="modal-title" id="proxyFormModalLabel">Add Proxy</h4>
+          <h4 class="modal-title" id="proxyFormModalLabel">
+            {{_ 'proxyForm_header_text' }}
+          </h4>
         </div>
-        {{ # autoForm collection=proxiesCollection id="proxyForm" type=formType doc=proxy}}
+        {{# autoForm
+          collection=proxiesCollection
+          id="proxyForm"
+          type=formType doc=proxy}}
```
Comment (marla-singer) [inline proxies/client/form/form.html]: Separate this line to two lines too

File: apis/client/view/settings/settings.html (hunk 2 above)
Comment (nnn) [inline apis/client/view/settings/settings.html]: Since it in settings view now, we don't need this check anymore.

File: apis/client/import_documentation/import.html (hunk 1 above)
Comment (brylie) [inline apis/client/import_documentation/import.html]: This is fixed in a later commit. Please pull recent changes.

File: proxies/client/form/form.html (hunk 8 above)
Comment (brylie) [inline proxies/client/form/form.html]: This is already done.

File: feedback/client/list/list.html (hunk 7 above)
Comment (brylie) [inline feedback/client/list/list.html]: Done.

File: documentation/client/manage/manage.html (hunk 5 above)
Comment (brylie) [inline documentation/client/manage/manage.html]: Fixed.
